automatic/keeweb/update.ps1

```diff
@@ -22,48 +20,36 @@ function global:au_SearchReplace {
 
 function global:au_GetLatest {
 
-    $downloadedPage = Invoke-WebRequest -Uri $releases -UseBasicParsing
-
-    $baseUrl = $([System.Uri]$releases).Authority
-    $scheme = $([System.Uri]$releases).Scheme
-
-    $url32 = $downloadedPage.links | ? href -match '32.exe$' | select -First 1 -expand href
-    if ($url32.Authority -cnotmatch $baseUrl) {
-        $url32 = $scheme + '://' + $baseUrl + $url32
-    }
-    $url32SegmentSize = $([System.Uri]$url32).Segments.Length
-    $filename32 = $([System.Uri]$url32).Segments[$url32SegmentSize - 1]
-
-    $url64 = $downloadedPage.links | ? href -match '64.exe$' | select -First 1 -expand href
-    if ($url64.Authority -cnotmatch $baseUrl) {
-        $url64 = $scheme + '://' + $baseUrl + $url64
+    $jsonAnswer = (Invoke-WebRequest -Uri "https://api.github.com/repos/keeweb/keeweb/releases/latest" -UseBasicParsing).Content | ConvertFrom-Json
```

Comment:
As mentioned in the other PR you opened, this will cause issues eventually as it is used without an API token.
Rate limiting will be hit quite often.
Please change to use the web URL, or use it with an API token.

Reply:
Taken care of.